com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkList.cs

```diff
@@ -374,7 +374,7 @@ public IEnumerator<T> GetEnumerator()
         public void Add(T item)
         {
             // check write permissions
-            if (!CanClientWrite(m_NetworkBehaviour.NetworkManager.LocalClientId))
+            if (m_NetworkBehaviour && !CanClientWrite(m_NetworkBehaviour.NetworkManager.LocalClientId))
```

Comment:
@PitouGames 
What this effectively does is open the possibility that an instance can write to the NetworkList even if it doesn't have permissions which upon spawning the authority's values for this NetworkList will overwrite any values set and that could lead to user confusion as to why the values aren't being applied.  
(same for all cases below)

It is a good idea... just need to noodle over it a bit more. 👍

Reply:
Generally speaking, this was sort of why we allow for the assignment of values upon instantiating the instance... which is problematic if you want to expose it as a property in the inspector view...and that too can get overwritten.
_(still pondering)_

Reply:
Yes, it opens the possibility that an instance writes to the NetworkList even if it won't have permissions when spawned, and this is what I need.

The issue is that today, doing an operation on the list throws an uninformative `NullReferenceException` instead of showing the warning you added specially for this case:
![image](https://user-images.githubusercontent.com/22180158/235377413-4d80ad7f-5f62-4efa-a812-8784a384dabb.png)